org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/Platform.java

```diff
@@ -0,0 +1,182 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.lsp4xml;
+
+/**
+ * PLatform information about OS and JVM.
+ */
+public class Platform {
+
+	private static final OS os = new OS();
+
+	private static final JVM jvm = new JVM();
```

Comment:
no need to keep it static, as it should be used once on startup

Reply:
I agree for JVM, but for os it used to initialize isWindows static variable

Reply:
yes, that's why I only commented for JVM